colony/branch_utils.py

```diff
@@ -180,7 +180,7 @@ def wait_and_then_delete_branch(sb_manager: SandboxesManager, sandbox_id, repo,
         else:
             time.sleep(10)
             BaseCommand.message(
-                f"Still waiting for sandbox (id={sandbox_id}) to prepare artifacts..."
+                f"Waiting for the Sandbox ({sandbox_id}) to launch..."
```

Comment:
Yeah I didn't want to get to the implementation details (waiting for artifacts - or maybe tomorrow they'll be another stage we'll need to wait for) so I just used 'Launch' in a generic way

Reply:
ok, I understand. But I think we need to use something else and not 'Launch' because when you create a sandbox it says in the UI 'Launching'. before it becomes active. So the word 'launch' is confusing here IMHO.
Maybe say something like "waiting for sandbox to start deployment"?

Reply:
How about just 'Starting the Sandbox' and then 'Sandbox still starting.....'

Reply:
This works for me

Reply:
@ronenquali I think this is the last thing to resolve on this PR